boards/amd/versal2_rpu/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+
+if(EXISTS "${BOARD_DIR}/${BOARD}-qemu.dts")
```

Comment:
why would the file not exist?

Reply:
This is generic rule and this is generic Versal Gen 2 board but under the same folder specific boards can be added. We have it internally for covering tests which are not possible to emulate on QEMU. That's why rule is taken when that file exists only.

Reply:
Note that files should not be overlayed in the folder, boards should instead be extended https://docs.zephyrproject.org/latest/hardware/porting/board_porting.html#board-extensions

This part needs 2 space indent inside the if()

Reply:
ad Note) I can't see any upstream board which uses it. (git grep "extend:" upstream/main) but will look at.

ad 2 space) yes this should be fixed. Thanks.

Reply:
No upstream board would use it in-tree because it is for out of tree usage only but it is used out of tree by several users

Reply:
Good to know thanks.